src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -0,0 +1,200 @@
+import Connector from '../engine/Connector.mjs';
+const EXIF = require('exif-js');
+
+export default class VizShonenJump extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'vizshonenjump';
+        super.label = 'Viz - Shonen Jump';
+        this.tags = [ 'manga', 'english' ];
+        this.url = 'https://www.viz.com/shonenjump';
```

Comment:
Do not not append the path here. I'm not sure if and how this will affect copy & paste feature since a pasted manga URL such as https://www.viz.com/one-piece may not match `this.url => https://www.viz.com/shonenjump`...
Better adding the path in `_getMangas()`